releases/0.9.0-incubating/INSTALL.md

```diff
@@ -139,18 +144,21 @@ remains clueless, please [log an issue](https://github.com/apache/incubator-open
 
 ## Deploy OpenWhisk
 
-Stay under the directory of incubator-openwhisk-<version>, and run the following ansible scripts one by one:
+We recommend you set the environment variable $OPENWHISK_HOME on your local machine to the extracted directory incubator-openwhisk-<version>.
+The deployment of OpenWhisk can also work without setting this environment variable. It is UNACCEPTABLE to set $OPENWHISK_HOME
+to empty or a wrong directory.
+
+Make sure your terminal goes to the directory of incubator-openwhisk-<version>/ansible, and run the following ansible
+scripts one by one:
```

Comment:
If you’re going to use OPENWHISK_HOME as below for wsk property config then I suggest also 

cd $OPENWHSIK_HOME/ansible

Reply:
done